packages/jest-jasmine2/src/index.js

```diff
@@ -34,7 +34,9 @@ async function jasmine2(
     testPath,
   );
   const jasmineFactory = runtime.requireInternalModule(JASMINE);
-  const jasmine = jasmineFactory.create();
+  const jasmine = jasmineFactory.create({
+    testPath,
```

Comment:
Instead of adding this to Jasmine, could we instead patch the results object like we do for snapshots below on line 110?

See https://github.com/facebook/jest/pull/4594/files#diff-e4399d1682ddb42f8a43f1a917d42d87R110

Reply:
I'll give that a shot

Reply:
@cpojer so I think that code is what deals with jest reporters not with jasmine reporters. Jasmine reporters are what I'm trying to modify because currently jest reporters do not provide callbacks for when an individual test is going to be run. The lowest level offered for those are before a describe block is run.